pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -611,28 +610,13 @@ private boolean hasRemoteProducers() {
 
     public CompletableFuture<Void> startReplProducers() {
```

Comment:
This method can return void

Reply:
Yes, just keep the original signature.

Reply:
+1

Reply:
There are some methods like `checkReplication`,`addProducer` that invoke this method and use `thanAccept`.
So keep the original signature is much more better.
@AnonHxy @codelipenghui